genesis.toml

```diff
@@ -0,0 +1,19 @@
+version = 1
+timestamp = 1672531200
+
+[[accounts]]
+type = "BasicWallet"
+mode = "RegularAccountImmutableCode"
+account_seed = "a123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+auth_seed = "b123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+auth_scheme = "RpoFalcon512"
```

Comment:
A couple of comments:

- I'm not sure we should use `account_seed` here as `account_seed` has a slightly different meaning (i.e., it is a seed which we get after we create the account). The term we usually use for this is `init_seed` - but I think here it might be confusing. Maybe we just say `seed`.
- I would probably move `auth_seed` after `auth_scheme`.
- We usually prefix `hex` data with `0x` to make it clear that this is hex encoding.

Reply:
fixed